src-tauri/src/mesh/device/state.rs

```diff
@@ -316,3 +319,11 @@ impl MeshDevice {
         }
     }
 }
+
+impl MeshGraph {
+    pub fn update_graph(&mut self, device: MeshDevice) {
```

Comment:
Is `update_graph` a descriptive enough name for this? Not sure I get what this is doing

Reply:
renamed to `regenerate_graph_from_device_info`, and swapped the unwrap of the `meshdevice` for an `if let` destructuring pattern